Umweltzone/src/main/res/values-da/strings.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright (C) 2019  Tobias Preuss
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+
+    <!--Action Bar items-->
+    <string name="action_cities">Byer</string>
+    <string name="action_share_app">Del denne app &#8230;</string>
+    <string name="action_changelog">Versions information</string>
+
+    <!-- Share intent -->
+    <string name="share_intent_message">
+        <![CDATA[Vil du pr&#248;ve Umweltzone? Android appen har alt information omkring milj&#248; zoner og diesel fri zoner i Tyskland .
+        \nF&#229; appen her gratis: https://play.google.com/store/apps/details?id=de.avpptr.umweltzone]]>
```

Comment:
Here special characters do not have to be escaped. Otherwise the HTML-entity codes show up in the email.

Reply:
Fixed.